src/Installer.php

```diff
@@ -223,7 +223,7 @@ private function generateConcreteIndividualCode(EntityPackageBuilder $graph)
                 $this->writeIfVeryVerbose(
                     '        - Generating interface for <info>' . $entity->getName() . '</info>'
                 );
-                ReflectionGenerator::generateInIsolation($entity->getName());
+                ReflectionGenerator::main($entity->getName());
```

Comment:
Since it's no longer process isolated, I'd expect the ReflectionGenerator to be created in the [activate](https://github.com/hostnet/entity-plugin-lib/blob/master/src/Plugin.php#L16-L39) function of Plugin, which creates the whole dependency graph (and does no work). This should make the unit-tests a lot easier.